[debug] [2025-09-04T03:24:21.258Z] ----------------------------------------------------------------------
[debug] [2025-09-04T03:24:21.259Z] Command:       /usr/local/bin/node /Users/tzuhaoshan/.npm-global/bin/firebase deploy
[debug] [2025-09-04T03:24:21.260Z] CLI Version:   14.12.0
[debug] [2025-09-04T03:24:21.260Z] Platform:      darwin
[debug] [2025-09-04T03:24:21.260Z] Node Version:  v22.17.0
[debug] [2025-09-04T03:24:21.260Z] Time:          Thu Sep 04 2025 11:24:21 GMT+0800 (台北標準時間)
[debug] [2025-09-04T03:24:21.260Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-09-04T03:24:21.363Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-09-04T03:24:21.363Z] > authorizing via signed-in user (aa0956266386@gmail.com)
[debug] [2025-09-04T03:24:21.364Z] [iam] checking project auto-scheduler-system-b597a for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2025-09-04T03:24:21.364Z] Checked if tokens are valid: false, expires at: 1756909281754
[debug] [2025-09-04T03:24:21.364Z] Checked if tokens are valid: false, expires at: 1756909281754
[debug] [2025-09-04T03:24:21.364Z] > refreshing access token with scopes: []
[debug] [2025-09-04T03:24:21.365Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-09-04T03:24:21.365Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-04T03:24:51.405Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------7971cd01d386d2166f1d4b43","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[error] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-09-04T03:24:51.406Z] [iam] error while checking permissions, command may fail: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-09-04T03:24:51.407Z] Checked if tokens are valid: false, expires at: 1756909281754
[debug] [2025-09-04T03:24:51.407Z] Checked if tokens are valid: false, expires at: 1756909281754
[debug] [2025-09-04T03:24:51.407Z] > refreshing access token with scopes: []
[debug] [2025-09-04T03:24:51.408Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-09-04T03:24:51.408Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-04T03:25:21.413Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------724f465e8d212175538d0f30","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[error] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-09-04T03:25:21.415Z] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-09-04T03:25:26.426Z] Error: Timed out.
    at Timeout._onTimeout (/Users/tzuhaoshan/.npm-global/lib/node_modules/firebase-tools/lib/utils.js:272:49)
    at listOnTimeout (node:internal/timers:588:17)
    at process.processTimers (node:internal/timers:523:7)
[error] 
[error] Error: An unexpected error has occurred.
